src/test-gui/java/de/qabel/desktop/ui/about/AboutPage.java

```diff
@@ -0,0 +1,33 @@
+package de.qabel.desktop.ui.about;
+
+import com.sun.javafx.robot.FXRobot;
+import de.qabel.desktop.ui.AbstractPage;
+import org.testfx.api.FxRobot;
+
+import static org.junit.Assert.assertTrue;
+
+public class AboutPage extends AbstractPage {
+    private AboutController controller;
+
+    public AboutPage(FXRobot baseFXRobot, FxRobot robot, AboutController controller) {
+        super(baseFXRobot, robot);
+        this.controller = controller;
+    }
+
+    public void btnShowPopup() {
+        clickOn("#btnThanks");
+        setPopupText();
+        showAboutPopup();
+        waitUntil(() -> controller.popupController.textAreaPopup != null);
+        assertTrue(controller.popupController.aboutPopup.isVisible());
```

Comment:
Please move the assertion to the test.
The structure for the PageObject pattern is that the Page encapsulates GUI interactions and the Test can do it's assertions on it. This way the test is cleanly readable like 
assertTrue(page.showPopup().isVisible());
but the test is still independent of the gui in case of redesigns etc.